### ISSUE TITLE
Xiaomi sync blocked

### ISSUE THREAD

**Issue by rube200** [anudeepND/blacklist#198]
<div align="center">  
  <img width="280" height="70" src="https://raw.githubusercontent.com/anudeepND/blacklist/master/images/blacklist-logo.png" alt="logo" />
</div>

<h2 align="center">Issue Submit Form</h2>
<div align="center">
  Provide the following info properly, which will help me to resolve your issue quickly.
</div>

### Issue(s):
Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [ ] Whitelist
- [ ] Blacklist
- [x] App Broken
- [ ] Website Broken
- [ ] Request
- [ ] Other Issue

<hr>

### Domain(s):
If you are submitting this issue for whitelist/blacklist issue, send me the domain(s) for whitelisting/blacklisting here. Kindly use the **Code Tag** to prevent tracking.

- Domain(s):

```
us.micardapi.micloud.xiaomi.net
us.phonecallapi.micloud.xiaomi.net
us.smsapi.micloud.xiaomi.net
```

<hr>

### Details:
You can attach any screen shot or log of the issue or advert, this will help to highlight it.

- Your Issue Detail: 
Hello, I was looking at the gravity table of my pihole and I noticed with these addresses being blocked, so I went to my xiaomi cell phone and synced my data (everything except the gallery) and in this case the contacts, calls and sms did not synchronization, so I decided to manually whitelist these domains one by one and they were successful in synchronization after the whitelist.

**Comment by anudeepND** [comment]
Fixed in the latest update

**Comment by rube200** [comment]
Hi there i found more two, need i open a new issue?
```
us.micardapi.micloud.xiaomi.net
us.pdc.micloud.xiaomi.net
```
The first one is the contacts list sync from xiaomi cloud (contacts list sync fail without it)
And the second is used by calendar and assitent sync in xiaomi cloud

**Comment by anudeepND** [comment]
Thanks @rube200. Fixed in 621b5e243d82993911a66187a8c8145a9f734850
